django-tethne/models.py

```diff
@@ -0,0 +1,163 @@
+from django.db import models
+
+
+
+"""
+Model Name = Source
+
+Source represents the source of the Corpus
+Example 1. JSTOR, 2. ZOTERO or 3. WOS
+"""
+class Source(models.Model):
+    source_value = models.CharField(max_length=35, null=True)
+
+"""
+
+Model name = Corpus
+Will store the corpus details, for example source, the number of papers and label.
+"""
+
+class Corpus(models.Model):
+    source = models.ForeignKey(Source)
+    date_created = models.DateTimeField(auto_now=True, auto_created=True)
+    length = models.IntegerField(null=True)
+    label = models.CharField(max_length=10, null=True)
+
+
+"""
+Model name = Paper
+
+Each paper belongs to a Corpus. So, a foriegn key reference is created to the Corpus.
+
+"""
+
+class Paper(models.Model):
+    corpus = models.ForeignKey(Corpus)
+    pub_date = models.DateField(null=True)
+    is_collaborated = models.BooleanField()
```

Comment:
is_collaborated - This says whether the paper has a single author or multiple authors.